data/json/snippets/effect_on_conditions.json

```
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_1",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It is quite dense and appears to be several regular polyhedrons next to each other.",
+      "You see what seems to be a new constellation in the sky.  It appears to be a spiral with the central star constantly shifting color.",
+      "You see several shooting stars.  They seem larger than normal and are all a deep purple.",
+      "You see a shooting star.  It is bright red and does not fall in a straight line.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon, but square and green.",
+      "For a brief moment, the stars and moon all disappear.  When they come back, you're not sure if they're in the same places as they were."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_2",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It changes shape as you look from a triangle to a pentagon.",
+      "You see several shooting stars.  After they go below the horizon a bright glow briefly appears.",
+      "You see a shooting star.  It is bright red and stops falling after a second to hover there ominously.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon.  It almost looks like like an eye blinking.",
+      "You see a moving object in the sky with a blinking red light coming from it."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_3",
+    "text": [
+      "You see a once-familiar constellation and watch as several of its stars move.",
+      "You see what seems to be a new constellation in the sky.  It reminds you of a serpent - it moves like one.",
+      "You see a shooting star.  It seems to be heading straight for you before vanishing suddenly.",
+      "For a brief moment you see a pentagonal shape in the sky, roughly twice the size of the moon.  It splits into several triangles and then reforms before vanishing."
```

More dramatic effects for the last tier maybe ?
```suggestion
      "For a brief moment you see a pentagonal shape in the sky, roughly twice the size of the moon.  It splits into several triangles and then reforms before vanishing.",
      "As you take a look to the stars, they vanish and reappear instantly in the exact same, but slightly angled configuration. Three times.",
      "The sky is deep and dark ... as you notice it, the stars slowly populate back the night.",
      "A strange cloud seems to float at the very limit of your peripheral sight. As you look to it - in what should be its place in the night sky, evaporates a stain of light into distant shooting stars.",
      "For a short moment you're feeling like dragged to the side ... you did not move, the stars have.",
      "A large batch of dots of light disappear into the night. The city lights are powered anew and confusedly blink black into existence. Hold on ... you're looking at the sky."
```

Or maybe it is too much. Or it needs another tier.

---

```suggestion
      "For a brief moment, you see a pentagonal shape in the sky, roughly twice the size of the moon.  It splits into several triangles and then reforms before vanishing.",
      "As you look at the stars, they wink out for an instant, reappearing just slightly out of place, as if the night sky had been rotated ever so slightly. This occurs twice more, in rapid succession.
      "A strange cloud seems to float in the sky, at the very edge of your peripheral vision. Just as you shift your gaze to it, it evaporates, boiling away into a stream of shooting stars.",
      "For a brief moment, you feel like you've been jerked to the side. You haven't moved, though -- the stars have.",
      "As you look at the stars, a broad swath of them are plunged into darkness, then come back to life in ragged succession."
```

I removed the third tier, as its effect is replicated in the final one.

---

I think by "tier" Moltenhead is meaning the time after the Cataclysm for that level of effect on the sky (currently there are ones in this PR for 0, 14, and 50 IIRC).

---

Yep that's exactly what I meant by tier.